introduction/modules/modules.md

```diff
@@ -83,21 +86,21 @@ document. This does *not* generate an actual document, but writes the calls to t
 
 ### Local mail
 
-The local mail module is a stub implementation of the mail module. It is used as a local test implementation that 
-allows the application to start without having to provide an actual implementation. This does *not* send an actual 
-mail.
+The local email module is a stub implementation of the module called 'Mail'. It is used as a local test implementation 
```

Comment:
`The local mail module is a stub implementation of the 'mail' module. It is used as a local test implementation`

Again, the module is called mail.